libs/libcommon/src/libcommon/exceptions.py

```diff
@@ -99,6 +100,7 @@ def as_response(self) -> ErrorResponse:
     "ExternalServerError",
     "FeaturesError",
     "FeaturesResponseEmptyError",
+    "FileFormatMismatchBetweenSplits",
```

Comment:
I would suggest adding the word "Error" at the end, by convention.

Reply:
sure!

Reply:
done